build.gradle

```diff
@@ -195,7 +195,7 @@ subprojects {
     tasks.register('isolatedTest', Test) {
         maxParallelForks = 1
         useJUnitPlatform {
-            includeTags 'isolatedTest'
+            excludeTags 'isolatedTest'
```

Comment:
Can we boil our test tasks into 2 types, IMO it isn't very useful to separate slow tests from isolated tests - is there a good case that I'm missing out on?